### ISSUE TITLE
localhost is not opening

### ISSUE THREAD

**Issue by dzhoz0** [Rigellute/spotify-tui#927]
I installed spotify-tui through the AUR package, `spotify-tui`, when I finish authorization, the callback doesn't work the `localhost:8888` is not opened. Am I missing to start any init files or anything?

**Comment by 910249** [comment]
Are you sure on your spotify dashboard you put localhost:8888 + the callback uri? I had this bug a while ago, its easy to fix, just make sure your url is exactly as the documentation says it should be. 
https://github.com/Rigellute/spotify-tui#connecting-to-spotifys-api

**Comment by 910249** [comment]
Also if you think that AUR is the error try using snap to install it.

**Comment by dzhoz0** [comment]
> Are you sure on your spotify dashboard you put localhost:8888 + the callback uri?

Yes, I have already checked multiple times. However, my problem is not the invalid redirect uri, the problem is that the localhost:8888 is not running the computer.

> Try using snap to install it

Sorry, I can't stand using snap.

**Comment by 910249** [comment]
Hmm did you try reinstalling? Maybe the port is blocked?

**Comment by 910249** [comment]
I am not sure how to help you, I hope you find your answer.

**Comment by ashtrath** [comment]
still no update for this? i have the same issue here.
tried building from source same things happen, and no i'll not use snap.

**Comment by 910249** [comment]
Sorry I guess you will have to stick with the spotify ui until a fix happens

**Comment by xinaras** [comment]
Same problem here. I hope they fix this, I'm really into a CLI-based Spotify client.

**Comment by kardusenor** [comment]
Same problems here. Install with scoop on windows

**Comment by EllliesProjects** [comment]
is there a fix for this? it still won't open

**Comment by sewnie** [comment]
https://github.com/aome510/spotify-player

**Comment by slateeho** [comment]
use just http insted https, as described in

> Are you sure on your spotify dashboard you put localhost:8888 + the callback uri? I had this bug a while ago, its easy to fix, just make sure your url is exactly as the documentation says it should be. https://github.com/Rigellute/spotify-tui#connecting-to-spotifys-api

**Comment by thiagopuntar** [comment]
> use just http insted https, as described in
> 

A dumb thing but happened to me, add http in front of localhost 😅 .
